.github/ISSUE_TEMPLATE/Other_issue_template.md

```diff
@@ -0,0 +1,5 @@
+---
+name: Other advanced issue
```

Comment:
I just noticed GitHub uses the filename to order the options when opening an issue, not this display name. So, we can put any display text as long as the filename starts with a low-order letter.